tests/many_to_many/models.py

```diff
@@ -33,12 +33,21 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
+class Author(models.Model):
```

Comment:
If you want to keep the two tests together, maybe `tests/prefetch_related` is a better place as it seems there are models there that would work. Otherwise, I'd put the FK test in `tests/many_to_one` rather than adding a new model here (unless there's some reasoning that I'm missing?).